hawkbit-runtime/docker/docker-compose-stack.yml

```diff
@@ -31,6 +31,8 @@ services:
 
   # ---------------------
   # hawkBit simulator
+  # username: admin
```

Comment:
IMHO it would be enough to mention the default credentials in the documentation only :)

Reply:
Sure, made the change.